docs/siem/siem-ui.asciidoc

```diff
@@ -1,102 +1,169 @@
 
 [role="screenshot"]
-image::siem/images/detections-ui.png[]
-
-[float]
-[[cases-ui]]
-=== Cases (beta)
-
-Cases are used to open and track security issues directly in Elastic Security. 
-Cases list the original reporter and all users who contribute to a case
-(`participants`). Case comments support Markdown syntax, and allow linking to
-saved Timelines. Additionally, you can send cases to external systems from
-within Elastic Security.
-
-For information about opening, updating, and closing cases, see
-{security-guide}/cases-overview.html[Cases] in the Elastic Security Guide.
-
-[role="screenshot"]
-image::siem/images/cases-ui.png[]
-
-[float]
-[[timelines-ui]]
-=== Timeline
-
-Timeline is your workspace for threat hunting and alert investigations. 
-
-[role="screenshot"]
-image::siem/images/timeline-ui.png[Elastic Security Timeline]
-
-You can drag objects of interest into the Timeline Event Viewer to create
-exactly the query filter you need. You can drag items from table widgets within
-Hosts and Network pages, or even from within Timeline itself.
-
-A timeline is responsive and persists as you move through Elastic Security
-collecting data. 
+image::../siem/images/workflow.png[Elastic Security workflow]
+
+Here's an overview of the flow and its components:
+
+* Data is shipped from your hosts to {es} via beat modules and the Elastic
+Endpoint Security agent integration:
+** {security-guide}/install-endpoint.html[{es-sec-endpoint}] - Elastic agent integration that
+protects your hosts {security-guide}/detection-engine-overview.html#malware-prevention[against malware] and ships these data sets:
+***  *Windows*: Process, network, file, DNS, registry, DLL and driver loads,
+malware security detections
+*** *Linux/macOS*: Process, network, file
+** https://www.elastic.co/integrations?solution=security[Beat modules]: {beats}
+are lightweight data shippers. Beat modules provide a way of collecting and
+parsing specific data sets from common sources, such as cloud and OS events,
+logs, and metrics. Common security-related modules are listed {security-guide}/ingest-data.html#enable-beat-modules[here].
+* The {security-app} in {kib} is used to manage the *Detection engine*,
+*Cases*, and *Timeline*, as well as administer hosts running Endpoint Security:
+** Detection engine: Automatically searches for suspicious host and network
+activity via the following:
+*** {security-guide}/detection-engine-overview.html#detection-engine-overview[Detection rules]: Periodically search the data
+({es} indices) sent from your hosts for suspicious events. When a suspicious
+event is discovered, a detection alert is generated. External systems, such as
+Slack and email, can be used to send notifications when alerts are generated.
+You can create your own rules and make use of our {security-guide}/prebuilt-rules.html[prebuilt ones].
+*** {security-guide}/detections-ui-exceptions.html [Exceptions]: Reduce noise and the number of
```

Comment:
Think the space between the topic link and the `[Exceptions]` might need to be removed for the text to show up as linked text.